Can't loading rest.xargs sample ...

Bug on rest.xargs sample (and loading org.eclipse.concierge.service.rest.client-1.0.0.*.jar.

`Exception in thread "main" org.osgi.framework.BundleException: Syntactic error in bundle manifest
        at org.eclipse.concierge.compat.service.BundleManifestTwo.processManifest(BundleManifestTwo.java:116)
        at org.eclipse.concierge.BundleImpl$Revision.<init>(BundleImpl.java:1608)
        at org.eclipse.concierge.BundleImpl$JarBundleRevision.<init>(BundleImpl.java:3813)
        at org.eclipse.concierge.BundleImpl.readAndProcessInputStream(BundleImpl.java:399)
        at org.eclipse.concierge.BundleImpl.<init>(BundleImpl.java:228)
        at org.eclipse.concierge.Concierge.installNewBundle(Concierge.java:4253)
        at org.eclipse.concierge.Concierge.installNewBundle(Concierge.java:4216)
        at org.eclipse.concierge.Concierge$BundleContextImpl.installBundle(Concierge.java:4842)
        at org.eclipse.concierge.compat.service.XargsFileLauncher.processXargsInputStream(XargsFileLauncher.java:178)
        at org.eclipse.concierge.compat.service.XargsFileLauncher.processXargsFile(XargsFileLauncher.java:83)
        at org.eclipse.concierge.Concierge.doMain(Concierge.java:617)
        at org.eclipse.concierge.Concierge.main(Concierge.java:536)
Caused by: java.lang.IllegalArgumentException: invalid version "1.0.0..SNAPSHOT-20190910193934": invalid qualifier ".SNAPSHOT-20190910193934"
        at org.osgi.framework.Version.validate(Version.java:205)
        at org.osgi.framework.Version.<init>(Version.java:156)
        at org.osgi.framework.Version.valueOf(Version.java:257)
        at org.osgi.framework.Version.parseVersion(Version.java:228)
        at org.eclipse.concierge.compat.service.BundleManifestTwo.processManifest(BundleManifestTwo.java:110)
        ... 11 more`

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,5 @@
 language: java
+dist: trusty
 jdk:
   - oraclejdk8
 install:

--- a/bundles/org.eclipse.concierge.service.rest.client/build.gradle
+++ b/bundles/org.eclipse.concierge.service.rest.client/build.gradle
@@ -6,61 +6,61 @@ import de.undercouch.gradle.tasks.download.Download
 
 task fetchFromOrbit {
     if (! file("libs/org.restlet_2.0.5.v201605172130.jar").exists()) {
-		download {
-        	src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.restlet_2.0.5.v201605172130.jar'              
-         	dest 'libs'
-    	}
+        download {
+            src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.restlet_2.0.5.v201605172130.jar'              
+            dest 'libs'
+        }
     }
     if (! file("libs/org.json_1.0.0.v201011060100.jar").exists()) {
-		download {
-        	src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.json_1.0.0.v201011060100.jar'              
-         	dest 'libs'
-    	}
+        download {
+            src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.json_1.0.0.v201011060100.jar'              
+            dest 'libs'
+        }
     }    
 }
 
 compileJava.dependsOn(fetchFromOrbit)
 
 repositories {
-   flatDir {
-       dirs 'libs'
-   }
+    flatDir {
+        dirs 'libs'
+    }
 }
 
 dependencies {
-   compile name: 'org.json_1.0.0.v201011060100'
-   compile name: 'org.restlet_2.0.5.v201605172130'
-   compile name: 'org.restlet.ext.json'
-   compile name: 'org.restlet.ext.xml'
+    compile name: 'org.json_1.0.0.v201011060100'
+    compile name: 'org.restlet_2.0.5.v201605172130'
+    compile name: 'org.restlet.ext.json'
+    compile name: 'org.restlet.ext.xml'
 }
 
 sourceSets.main.resources { srcDirs += ['libs']; include "*.jar" }
 
 jar {
     manifest {
-		from('META-INF/MANIFEST.MF') {
-			eachEntry { details ->
-				if (details.key == 'Bundle-Version') {
-                   	details.value = details.value.replace ("qualifier", rootProject.buildQualifier)
-   	            }
-   	            if (details.key == 'Bundle-ClassPath') {
-   	            	details.value = details.value.replace ('libs/', '')
-   	            }
-       		}
-       	}
-	}
+        from('META-INF/MANIFEST.MF') {
+            eachEntry { details ->
+                if (details.key == 'Bundle-Version') {
+                    details.value = details.value.replace (".qualifier", rootProject.buildQualifier)
+                }
+                if (details.key == 'Bundle-ClassPath') {
+                    details.value = details.value.replace ('libs/', '')
+                }
+            }
+        }
+    }
 }
 
 publishing {
     publications {
-   	    mavenJava(MavenPublication) {
-	   	    groupId 'org.eclipse.concierge'
+        mavenJava(MavenPublication) {
+            groupId 'org.eclipse.concierge'
             artifactId 'org.eclipse.concierge.service.rest.client'
             version version
-			pom {
-				name = 'Eclipse Concierge'
-				description = 'REST Client Service'
-			}
-       	}
-   	}
+            pom {
+                name = 'Eclipse Concierge'
+                description = 'REST Client Service'
+            }
+        }
+    }
 }

--- a/bundles/org.eclipse.concierge.service.rest/.classpath
+++ b/bundles/org.eclipse.concierge.service.rest/.classpath
@@ -5,7 +5,7 @@
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="lib" path="libs/org.restlet.ext.json.jar"/>
 	<classpathentry kind="lib" path="libs/org.restlet.ext.xml.jar"/>
-	<classpathentry kind="lib" path="/Users/rjan/git/concierge/bundles/org.eclipse.concierge.service.rest/libs/org.restlet_2.0.5.v201605172130.jar"/>
-	<classpathentry kind="lib" path="/Users/rjan/git/concierge/bundles/org.eclipse.concierge.service.rest/libs/org.json_1.0.0.v201011060100.jar"/>
+	<classpathentry kind="lib" path="libs/org.restlet_2.0.5.v201605172130.jar"/>
+	<classpathentry kind="lib" path="libs/org.json_1.0.0.v201011060100.jar"/>
 	<classpathentry kind="output" path="bin"/>
 </classpath>

--- a/bundles/org.eclipse.concierge.service.rest/build.gradle
+++ b/bundles/org.eclipse.concierge.service.rest/build.gradle
@@ -6,15 +6,15 @@ import de.undercouch.gradle.tasks.download.Download
 
 task fetchFromOrbit {
     if (! file("libs/org.restlet_2.0.5.v201605172130.jar").exists()) {
-		download {
-        	src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.restlet_2.0.5.v201605172130.jar'              
-         	dest 'libs'
-    	}
+        download {
+            src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.restlet_2.0.5.v201605172130.jar'              
+            dest 'libs'
+        }
     }
     if (! file("libs/org.json_1.0.0.v201011060100.jar").exists()) {
-		download {
-        	src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.json_1.0.0.v201011060100.jar'              
-         	dest 'libs'
+        download {
+            src 'http://download.eclipse.org/tools/orbit/downloads/drops/R20170303204511/repository/plugins/org.json_1.0.0.v201011060100.jar'              
+            dest 'libs'
     	}
     }    
 }
@@ -38,29 +38,29 @@ sourceSets.main.resources { srcDirs += ['libs']; include "*.jar" }
 
 jar {
     manifest {
-		from('META-INF/MANIFEST.MF') {
-			eachEntry { details ->
-				if (details.key == 'Bundle-Version') {
-                   	details.value = details.value.replace ("qualifier", rootProject.buildQualifier)
-   	            }
-   	            if (details.key == 'Bundle-ClassPath') {
-   	            	details.value = details.value.replace ('libs/', '')
-   	            }
-       		}
+        from('META-INF/MANIFEST.MF') {
+            eachEntry { details ->
+                if (details.key == 'Bundle-Version') {
+                    details.value = details.value.replace (".qualifier", rootProject.buildQualifier)
+                }
+                if (details.key == 'Bundle-ClassPath') {
+                    details.value = details.value.replace ('libs/', '')
+                }
+            }
        	}
-	}
+    }
 }
 
 publishing {
     publications {
-   	    mavenJava(MavenPublication) {
-	   	    groupId 'org.eclipse.concierge'
+        mavenJava(MavenPublication) {
+            groupId 'org.eclipse.concierge'
             artifactId 'org.eclipse.concierge.service.rest'
             version version
-			pom {
-				name = 'Eclipse Concierge'
-				description = 'REST Service'
-			}
+            pom {
+                name = 'Eclipse Concierge'
+                description = 'REST Service'
+            }
        	}
-   	}
+    }
 }

